packages/app/src/analytic-data-tracker/services/analytic-data-tracker.service.ts

```diff
@@ -44,6 +44,13 @@ export class AnalyticDataTrackerService implements AnalyticDataTrackerClass {
     event: TrackEvent,
     externalUsageDeviceOnly = true
   ): Promise<AxiosResponse | undefined> {
+    console.log(
+      this.trackingEnabled,
+      externalUsageDeviceOnly,
+      this.externalUsageDevice,
+      event.e_c
+    )
+
```

Comment:
this seems legit, but still is console.log :) Should it stay here?

Reply:
resolved in 94eb632df33bbd05fa6bba41bcb9482d6d1cfcd5